docs/building-apps/governing-apps/tokenomics/sns-checklist.mdx

```diff
@@ -110,14 +110,19 @@ In addition to performing comprehensive testing of all dapp operations using the
 
 ## 3.2. Community discussion
-It’s strongly recommended that developers have a discussion in the forum with the community about the decentralization plans. It’s suggested to start a thread at least a couple of weeks before the NNS proposal is created. This will allow the community to learn about your plans, ask questions, and build trust in the SNS launch. The SNS initialization and decentralization swap will not start unless enough NNS neurons vote in favor of the proposal.
+It’s strongly recommended that developers have a discussion in the forum with the community about the decentralization plans. It’s suggested to start a thread at least a couple of weeks before the NNS proposal is created. Please use the [SNS DAO > SNS Launch Proposals](https://forum.dfinity.org/c/sns-dao/sns-launch-proposals) subcategory on the forum. This will allow the community to learn about your plans, ask questions, and build trust in the SNS launch. The SNS initialization and decentralization swap will not start unless enough NNS neurons vote in favor of the proposal.
 
 It is recommended to share:
 
-- [x] The init file that is planned to be used. You can start from a [template](https://github.com/dfinity/ic/blob/master/rs/sns/cli/test_sns_init_v2.yaml).
+- [x] The init file that is planned to be used. You can start from a [template](https://github.com/dfinity/ic/blob/master/rs/sns/cli/test_sns_init_v2.yaml). Make sure you add comments providing context for the values specified. E.g. elaborate on the specific dapp canisters or the distribution of developer neurons.
 - [x] The whitepaper with a full description of the decentralization and tokenomics.
 - [x] Technical decomposition of the dapp architecture in terms of canisters, source code, and documentation so that the community can validate that the dapp will actually be a decentralized application after the swap.
-- [x] Explanation of the extent to which security reviews were considered relevant for the dapp and what kind of security reviews have been conducted for the dapp.
+- [x] Explanation of the extent to which security reviews were considered relevant for the dapp and what kind of security reviews have been conducted for the dapp, core DAO functionality open-sourced.
+- [x] Clearly articulate the minimum and maximum funding targets and describe the planned usage of the funds raised.
+- [x] Demonstrate that you have a working product with product-market fit and community traction
+- [x] Clear articulation and analysis of token distribution, vesting periods, voting power allocation at genesis (including rationale for dev and seed independence), its evolution over time, and justification of DAO decentralization.
+- [x] Clear definition of token utility, including elaboration on token value over time, income streams, treasury usage, and assessment via [SNS Tokenomics Analyzer](https://dashboard.internetcomputer.org/sns/tokenomics).
```

Comment:
Doesn't the tokenomics analyzer consider more the decentralization and token distribution after launch?
if I remember this correctly, maybe it would better fit to add this link on the bullet point above?